client/src/components/Landing.js

```diff
@@ -1,12 +1,31 @@
 import React, { Component, Fragment } from 'react';
+import * as S from './Landing.style';
+import { LOGIN_URL, SIGNUP_URL } from '../constants';
 import CHeader from './CHeader';
 
 class Landing extends Component {
+  login = () => {
+    const { history } = this.props;
+    history.push(LOGIN_URL);
+  };
+
+  signup = () => {
+    const { history } = this.props;
+    history.push(SIGNUP_URL);
+  };
+
   render() {
     return (
       <Fragment>
         <CHeader />
-        <h1>This is a h1</h1>
+        <S.Main>
+          <S.H1>MY</S.H1>
+          <S.H2>
+            Business <br /> launchpad
```

Comment:
We can probably find a way to separate these so that we don't have to use <br /> just for best practices :)